### ISSUE TITLE
Local webserver error, empty output

### ISSUE THREAD

**Issue by andyreagan** [andyreagan/phantom-crowbar#1]
Hi Andy, 

I've come across your phantom-crowbar script on your website and it's proving really useful to me, thanks for making it available on GitHub! I have one problem though which I wondered if you've experienced? I can't seem to extract SVG from webpages which I'm generating on a local webserver (so the address is `http://localhost:8000/../blah.html`).  When I try to do this I get the following response:

```
TypeError: null is not an object (evaluating 'svg.setAttribute')

  phantomjs://webpage.evaluate():32

  phantomjs://webpage.evaluate():55

Evaluated our code
```

Also the output file is empty.  I'm new to Javascript and D3 and after spending sometime looking in to this I still can't figure out why there should be this problem when the SVG is being generated on a local webserver.  Do you have any ideas?

Thanks very much for your time and thanks again for making the code available!

-Angela

**Comment by andyreagan** [comment]
Seems to me like the `selector` that you passed in on the call to phantom-crowbar didn't work.

If you can open up the console on the webpage you're trying to get the SVG from, and try running `document.getElementById("selector")`, make sure it's actually selecting the HTML that contains the SVG you want to take out.

**Comment by AngelaO** [comment]
Hi Andy, 
Thanks for getting back to me so quickly! You were right, because I was setting the SVG "id" attribute within a function in my script to generate a D3 image on a local webserver it wasn't set properly and `document.getElementById("mysvg")` returned `null` .  Once I set that attribute outside of the function `document.getElementById("mysvg")` no longer returned `null` and it works fine now! Thanks again for this code, so so useful and I tried a lot of sub-standard approaches before I found this one! :+1:
